samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/HouseTokenTransferManager.kt

```diff
@@ -0,0 +1,542 @@
+                    )
+                } else {
+                    id = AssetManager.createAssetPledge(
+                        rpc.proxy,
+                        localNetworkId!!,
+                        importNetworkId!!,
+                        params[0],      // Type
+                        params[1],      // ID
+                        recipientCert,
+                        nTimeout,
+                        "com.cordaSimpleApplication.flow.RetrieveStateAndRef",
+                        AssetContract.Commands.Delete(),
+                        issuer,
+                        obs
+                    )
+                }
+                println("Asset Pledge State created with contract ID ${id}.")
+            } catch (e: Exception) {
+              println("Error: ${e.toString()}")
+            } finally {
+                rpc.close()
+            }
+        }
+    }
+}
+
+/**
+ * Command to fetch the certificate (in base64) of the party owning the node.
+ */
+class FetchCertBase64HouseCommand : CliktCommand(name="get-cert-base64", help = "Obtain the certificate of the party owning a node in base64 format.") {
+    val config by requireObject<Map<String, String>>()
+    override fun run() = runBlocking {
+
+        val rpc = NodeRPCConnection(
+            host = config["CORDA_HOST"]!!,
+            username = "clientUser1",
+            password = "test",
+            rpcPort = config["CORDA_PORT"]!!.toInt())
+        try {
+            val certBase64 = rpc.proxy.startFlow(::GetOurCertificateBase64).returnValue.get()
+            println("Certificate in base64: $certBase64")
+        } catch (e: Exception) {
+            println("Error: ${e.toString()}")
+        } finally {
+            rpc.close()
+        }
+    }
+}
+
+/**
+ * Command to fetch the name of the party owning the node.
+ */
+class FetchPartyNameHouseCommand : CliktCommand(name="get-party-name", help = "Obtain the name of the party owning a node in the Corda network.") {
+    val config by requireObject<Map<String, String>>()
+    override fun run() = runBlocking {
+
+        val rpc = NodeRPCConnection(
+            host = config["CORDA_HOST"]!!,
+            username = "clientUser1",
+            password = "test",
+            rpcPort = config["CORDA_PORT"]!!.toInt())
+        try {
+            val partyName = rpc.proxy.startFlow(::GetOurIdentity).returnValue.get()
+            println("Name of the party owning the Corda node: $partyName")
+        } catch (e: Exception) {
+            println("Error: ${e.toString()}")
+        } finally {
+            rpc.close()
+        }
+    }
+}
+
+/*
+ * Populates the file 'networkID'+'_UsersAndCerts.json' with the users and their certificates.
+ * This is used during Pledge to get the recipientCert, and during Claim to get the pledgerCert.
+ */
+class SaveUserCertToFileHouseCommand : CliktCommand(name="save-cert", help = "Populates the file 'networkId' + '_UsersAndCerts.json' with the certificate of 'ourIdentity'")
```

Comment:
This and above two commands, I don't see they are related to House Token App or Simple App, they seem to common, can we move them outside these groups (subcommands)?

Reply:
With the new method, I think we can remove these. And have one common client commands at root level.

Reply:
Done.